### PR TITLE
Update a section for how to run a job locally

### PR DESCRIPTION
**Pull request by krzyzacy**
As people are asking. Improved from my demo session notes.

### CODE REVIEW

**Comment by xiangpengzhao** [comment]
Hooray! I love it!

File: jenkins/README.md [hunk 1]
```diff
@@ -37,3 +37,29 @@ Say you want to add a new job, foo.
...
+Below paths are from the root of the `kubernetes/test-infra` repo.
+
+The straight forward way to mimic a job run on Jenkins is, for example:
+```
+./jenkins/bootstrap.py --job=ci-kubernetes-e2e-gce-canary --json=1 --bare --timeout=40
+```
+
+If you want to upload the log and display in gubernator, you can append
+```
+--service-account=/path/to/service-account-cred.json --upload='gs://your-bucket-name/logs'
```
Comment (fejta) [inline jenkins/README.md]: May want to add a link to how to create a service account

Comment (krzyzacy) [inline jenkins/README.md]: done.

**Pull request merged**